src/features/campaigns/components/CampaignActionButtons.tsx

```diff
@@ -84,6 +87,15 @@ const CampaignActionButtons: React.FunctionComponent<
   };
 
   const menuItems = [
+    {
+      icon: <Map />,
+      label: messages.linkGroup.createCanvassAssignment(),
+      onClick: () =>
+        createCanvassAssignment({
+          campaign_id: campaign.id,
+          title: null,
+        }),
+    },
```

Comment:
Not sure it should be the first option in the list, but hey, we're experimenting.